Video.js

```diff
@@ -225,13 +225,13 @@ export default class Video extends Component {
 
     let nativeResizeMode;
     if (resizeMode === VideoResizeMode.stretch) {
-      nativeResizeMode = NativeModules.UIManager.RCTVideo.Constants.ScaleToFill;
+      nativeResizeMode = NativeModules.UIManager.getViewManagerConfig('RCTVideo').Constants.ScaleToFill;
```

Comment:
This will break old react native should use wrapper function. Example implementation https://github.com/react-native-community/react-native-webview/commit/263fc5ec3c4216e94ebba076a2d8ea7559e96f6d

Reply:
just made changes you suggested, thanks for the reference.